src/Caliburn.Micro.Platform/Platforms/netcore-avalonia/BootstrapperBase.cs

```diff
@@ -119,6 +127,16 @@ protected virtual void Configure()
         {
         }
 
+        /// <summary>
+        /// Override this to catch unhandled exceptions.
+        /// </summary>
+        /// <param name="sender">The sender.</param>
+        /// <param name="e">The event args.</param>
+        protected virtual void OnUnhandledException(object sender, DispatcherUnhandledExceptionEventArgs e)
```

Comment:
Will it ever be called? I can't see a call to this within this PR. Also why is this only added for the `netcore-avalonia` platform?

Reply:
In WPF, this method subscribes to the  the Application class DispatcherUnhandledException, is there something similar in Avalonia?

Reply:
removing avalonia ui does not have the DisplatcherUnhandledException